project-reports/2023/2023-Q2-Hyperledger-Bevel.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Bevel
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+-	Hyperledger Bevel continues to grow strong in terms of community interest.
+-	The project is on track with its roadmap and release commitments.
+
+# Questions/Issues for the TSC
+
+-	None at the moment
+
+# Releases
+
+*	Up comming end of May release [v0.14.0](https://github.com/hyperledger/bevel/projects/85)
+
+# Overall Activity in the Past Quarter
+
+1.	New platform added: Support for **substrate** based DSCP platform.
+2.	Major features:
+	-	**Cactus-Connectors** for HL Besu platform.
```

Comment:
Curious what this is. Can you point to documentation? Or discuss how Cactus and Bevel are working together?

Reply:
I'm also curious. There are Ethereum connectors within Cactus (Cacti), and I think the right home for any new "connectors" is the Cacti repository. But I'm happy to discuss this further with Bevel and Cacti maintainers.

Reply:
The documentation is pending; essentially, Bevel can now be used to deploy Cacti Connector for Besu and Fabric (although Fabric connector is not tested because issue with the connector itself). To deploy a connector, the `cactus_connector: enabled` has to be set for peers/nodes.